string type_speech
string text
string data